internal/cmd/ske/credentials/complete-rotation/complete_rotation.go

```diff
@@ -0,0 +1,117 @@
+)
+
+const (
+	clusterNameArg = "CLUSTER_NAME"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ClusterName string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("complete-rotation %s", clusterNameArg),
+		Short: "Completes the rotation of the credentials associated to a SKE cluster",
+		Long: fmt.Sprintf("%s\n%s\n%s\n%s\n%s\n\n%s\n%s\n%s\n%s",
+			"Completes the rotation of the credentials associated to a STACKIT Kubernetes Engine (SKE) cluster.",
+			"To ensure continued access to the Kubernetes cluster, please update your kubeconfig service account to the newly created account.",
+			"This is step 1 of a 2-step process to rotate all SKE cluster credentials. Tasks accomplished in this phase include:",
+			"  - The old certification authority will be dropped from the package.",
+			"  - The old signing key for the service account will be dropped from the bundle.",
+			"If you haven't, please start the process by running:",
+			"  $ stackit ske credentials start-rotation my-cluster",
+			"Generate a new kubeconfig file by running:",
+			"  $ stackit ske kubeconfig create my-cluster"),
+		Args: args.SingleArg(clusterNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Complete the rotation of the credentials associated to the SKE cluster with name "my-cluster"`,
+				"$ stackit ske credentials complete-rotation my-cluster"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to complete the rotation of the credentials for SKE cluster %q?", model.ClusterName)
+				err = confirm.PromptForConfirmation(cmd, prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			_, err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("complete rotation of SKE credentials: %w", err)
+			}
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(cmd)
+				s.Start("Completing credentials rotation")
+				_, err = wait.CompleteCredentialsRotationWaitHandler(ctx, apiClient, model.ProjectId, model.ClusterName).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for completing SKE credentials rotation %w", err)
+				}
+				s.Stop()
+			}
+
+			operationState := "Rotation of credentials is completed"
+			if model.Async {
+				operationState = "Triggered completion of credentials rotation"
+			}
+			cmd.Printf("%s for cluster %q\n", operationState, model.ClusterName)
+			cmd.Printf("Consider updating your kubeconfig with the new credentials, create a new kubeconfig by running:\n  $ stackit ske kubeconfig create %s\n", model.ClusterName)
```

Comment:
At this point the user should have already updated their kubeconfigs, right? Because the old credentials are no longer working after the complete rotation

Reply:
So I would say we could either phrase it like "If you haven't yet, ...` or maybe remove this altogether

Reply:
If we go with the first one, this would be another candidate for the (future) "warning" verbosity print

Reply:
This would be an important warning to have, probably before prompting for confirmation, similar to the portal where you see that warning before pressing "Continue"

Reply:
I don't think this is a big concern for a CLI user, they can always get the new kubeconfig with the CLI command. It isn't a "No going back" situation, access can always be provided at any point. My thinking with making it a suggestion at the end of the completion step is to serve as a reminder more than necessarily a warning, specially since there are legitimate use cases where the user may not even want to have kubectl access to the cluster.

Reply:
Ok it makes sense, then let's keep it at the end and make it a warning once the new verbosity pkg is in place